modules/generic_worker/templates/generic-worker.config.erb

```diff
@@ -30,16 +26,16 @@
   "shutdownMachineOnInternalError": false,
   "taskclusterProxyExecutable": "/usr/local/bin/taskcluster-proxy",
   "taskclusterProxyPort": 8080,
-  "openpgpSigningKeyLocation": "<%= @opengpg_signing_key %>",
   "ed25519SigningKeyLocation": "<%= @ed25519_signing_key %>",
-  "subdomain": "taskcluster-worker.net",
   "tasksDir": "<%= @task_dir %>",
   "workerGroup": "<%= @worker_group %>",
   "workerId": "<%= @hostname %>",
   "workerType": "<%= @worker_type %>",
+  "wstAudience": "taskcluster-net",
+  "wstServerURL": "https://websocktunnel.tasks.build",
   "workerTypeMetadata": {
     "machine-setup": {
-      "config": "https://github.com/mozilla-platform-ops/ronin_puppet/raw/master/modules/generic_worker/templates/generic-worker.config.erb",
+      "config": "https://github.com/mozilla-releng/build-puppet/raw/master/modules/generic_worker/templates/generic-worker.config.erb",
```

Comment:
Why change this back to the old config in build-puppet? it should be point to here.